pandas/tests/io/parser/common/test_ints.py

```diff
@@ -110,6 +115,41 @@ def test_integer_overflow_bug(all_parsers, sep):
     tm.assert_frame_equal(result, expected)
 
 
+@skip_pyarrow
+def test_integer_overflow_with_user_dtype(all_parsers, any_int_dtype):
+    dtype = any_int_dtype
+    parser = all_parsers
+
+    pdtype = pandas_dtype(any_int_dtype)
+    iinfo = np.iinfo(pdtype.type if is_extension_array_dtype(dtype) else pdtype)
+
+    for x in [iinfo.max, iinfo.min]:
```

Comment:
Could you use `pytest.mark.parameterize` for these for loop cases?

Reply:
Done